dwitter/templates/dweet/dweet.html

```diff
@@ -53,7 +93,7 @@
           parent.postMessage({
             'type': 'error',
             'error': ""+e,
-            'location': window.location.href
+            'location': location.href
```

Comment:
Why was this change made?

Reply:
I think I was just confused at one point and checked whether the window still contained the same things as the global scope, then forgot to change it back.